tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2222,6 +2223,37 @@ describe('scalar card', () => {
           }),
         ]);
       }));
+
+      it('toggles selected time when single fob is deselected', fakeAsync(() => {
+        const runToSeries = {
+          run1: [buildScalarStepData({step: 10})],
+          run2: [buildScalarStepData({step: 20})],
+          run3: [buildScalarStepData({step: 30})],
+        };
+        const dispatchedActions: Action[] = [];
+        spyOn(store, 'dispatch').and.callFake((action: Action) => {
+          dispatchedActions.push(action);
+        });
+        provideMockCardRunToSeriesData(
+          selectSpy,
+          PluginType.SCALARS,
+          'card1',
+          null /* metadataOverride */,
+          runToSeries
+        );
```

Comment:
Feel like line 2228-2243 is the same for every test in this "describe" and is going to be the same for future tests in this block, considering move them to beforeEach?